{
  "comments": [
    {
      "key": {
        "uuid": "85112316_e6a744aa",
        "filename": "ansible/roles/kubelet/templates/kubelet_healthcheck.sh",
        "patchSetId": 4
      },
      "lineNbr": 28,
      "author": {
        "id": 183
      },
      "writtenOn": "2019-11-28T15:56:02Z",
      "side": 1,
      "message": "can you print the exact error message too from the stack? if this ever happens, it will be impossible to debug the root cause",
      "revId": "993f422f03a73ad7ab78a60041da219cda4bbf37",
      "serverId": "4cada01b-0314-4fe0-84a1-f9ddc89863f9",
      "unresolved": true
    }
  ]
}